derp/derp_server.go

```diff
@@ -173,22 +177,22 @@ type Server struct {
 	mu       sync.Mutex
 	closed   bool
 	netConns map[Conn]chan struct{} // chan is closed when conn closes
-	clients  map[key.NodePublic]*clientSet
+	clients  map[NodeHandle]*clientSet
 	watchers set.Set[*sclient] // mesh peers
 	// clientsMesh tracks all clients in the cluster, both locally
 	// and to mesh peers.  If the value is nil, that means the
 	// peer is only local (and thus in the clients Map, but not
 	// remote). If the value is non-nil, it's remote (+ maybe also
 	// local).
-	clientsMesh map[key.NodePublic]PacketForwarder
+	clientsMesh map[NodeHandle]PacketForwarder
 	// peerGoneWatchers is the set of watchers that subscribed to a
```

Comment:
can we distinguish "set" in terms of outter map value, and not outter map in the docs here? i.e. the outermap key is the key of interest, the outermap value is the "set" in this context